portal/app/[locale]/_components/navbar/_components/help.tsx

```diff
@@ -0,0 +1,223 @@
+import { useNetworkType } from 'hooks/useNetworkType'
+import { useOnClickOutside } from 'hooks/useOnClickOutside'
+import { usePathnameWithoutLocale } from 'hooks/usePathnameWithoutLocale'
+import { useRouter } from 'i18n/navigation'
+import { locales } from 'i18n/routing'
+import { useLocale, useTranslations } from 'next-intl'
+import { ComponentProps, ReactNode, RefObject, useState } from 'react'
+
+import { CmcAttribution } from './cmcAttribution'
+import { TermsAndConditions } from './termsAndConditions'
+
+type Selectable = { selected?: boolean }
+type LanguageProps = {
+  active: string
+}
+type Props = {
+  event?: AnalyticsEventsWithChain
+  icon?: ReactNode
+  text: string
+  value?: string
+}
+
+const IconContainer = ({ children }: { children: ReactNode }) => (
+  <div
+    className="flex h-10 w-10
+      items-center justify-center rounded-lg bg-neutral-50
+      md:h-4 md:w-4"
+  >
+    {children}
+  </div>
+)
+
+const Row = (props: { children: ReactNode } & ComponentProps<'div'>) => (
+  <div
+    className="flex h-14 items-center justify-between px-4 md:h-fit"
+    {...props}
+  />
+)
+
+const MenuContainer = ({
+  children,
+  isOpen,
+  refProp,
+  ...props
+}: { children: ReactNode } & {
+  isOpen: boolean
+  refProp: RefObject<HTMLDivElement>
+} & ComponentProps<'div'>) => (
+  <div
+    {...props}
+    className={`w-full cursor-pointer rounded-lg transition-colors duration-300 md:py-2 ${
+      isOpen ? 'rounded-lg bg-neutral-50' : 'hover:bg-neutral-50'
+    }`}
+    ref={refProp}
+  >
+    {children}
+  </div>
+)
+
+const ItemText = ({
+  selected = false,
+  text,
+}: Pick<Props, 'text'> & Selectable) => (
+  <span
+    className={`text-base font-medium capitalize transition-colors duration-300
+      group-hover/item:text-neutral-950 md:text-sm ${
+        selected ? 'text-neutral-950' : 'text-neutral-700'
+      }`}
+  >
+    {text}
+  </span>
+)
+
+const ItemWithSubmenu = function ({
+  event,
+  icon,
+  subMenu,
+  text,
+  value,
+}: Props & { subMenu: ReactNode }) {
+  const [networkType] = useNetworkType()
+  const [isOpen, setIsOpen] = useState(false)
+  const { track } = useUmami()
+
+  const ref = useOnClickOutside<HTMLDivElement>(() => setIsOpen(false))
+
+  const onClick = function () {
+    setIsOpen(!isOpen)
+    track?.(event, { chain: networkType })
+  }
+
+  return (
+    <MenuContainer isOpen={isOpen} refProp={ref}>
+      <Row onClick={onClick}>
+        <div className="flex items-center gap-x-4 md:gap-x-2">
+          <IconContainer>{icon}</IconContainer>
+          <ItemText selected={isOpen} text={text} />
+        </div>
+        <div className="flex items-center gap-x-2">
+          <div className="hidden md:block">
+            <ItemText selected={true} text={value} />
+          </div>
+          <Chevron.Right />
+        </div>
+      </Row>
+      {isOpen && subMenu}
+    </MenuContainer>
+  )
+}
+
+const LanguageMenu = ({ active }: LanguageProps) => {
```

Comment:
Let's use the function style for non-inline functions, 

```suggestion
const LanguageMenu = function ({ active }: LanguageProps) {
```

so we can get rid of ` eslint-disable arrow-body-style */`

Reply:
Let's do all we can to avoid overriding linting rules. We shall stick to the rules... or change the rules.

Reply:
fixed